ios/MaasCore/Sources/MaasComponents/Supporting Files/Extensions/Text.swift

```diff
@@ -0,0 +1,23 @@
+public extension Text {
+
+    /**
+     Shortcut for ** • ** with whitetspaces symbol.
+     - parameters:
+        - spacedOut: Defines does dot symbol should by between whitespaces.
+     */
+    static func dot(spacedOut: Bool = false) -> Text {
```

Comment:
The character • is called a "bullet" afaik, do you think this may be confused with the actual "dot" .? 😄